assignment4/count_islands_test.py

```diff
@@ -0,0 +1,49 @@
+'''
+Created on 23 June 2017
+
+@author: Eva
+'''
+
+import unittest
+from count_islands import count_islands_rec, count_islands_iter
+
+class CountingIslandsTest(unittest.TestCase):
+
+    def setUp(self):
+        self.array = [[False, True, False, True], [True, True, False, False], [False, False, True, False], [False, False, True, False]]
```

Comment:
It is usually helpful to see the data you are testing on in your test code itself.
Instead of defining a bunch of constants in `setUp()`, you can define them where you use them.
It is a trade-off, of course, and might make sense to define the constants in some cases, especially when they are the same for all tests.